src/mapping_navigation/scripts/navigation.py

```diff
@@ -12,17 +13,19 @@ class Navigation:
     """
 
     def __init__(self):
-        self.steering_pub = rospy.Publisher('lka/steering', Float64, queue_size=10)
-        self.curve_pub = rospy.Publisher('curveData', String, queue_size=10)
+        self.steering_pub = rospy.Publisher('steering', Float64, queue_size=10)
+        self.curve_pub = rospy.Publisher('curve_data', String, queue_size=10)
```

Comment:
Minor thing we'll need to decide (though not urgent)- are we using camel case or snake case for topic names.

Reply:
yeah idk, not sure what is proper form